cpu/lm4f120/include/periph_cpu.h

```diff
@@ -115,6 +115,27 @@ typedef enum {
     ADC_RES_16BIT = 0xd00,            /**< not supported by hardware */
 } adc_res_t;
 
+
+/**
+ * @name SPI configuration
+ * @{
+ */
+#define SPI_NUMOF 4
```

Comment:
Sorry, I'm not sure to see the problem here. Is it a rule in RIOT to put even single token macro between `()` ?